### ISSUE TITLE
Show PRs that have commits after review

### ISSUE THREAD

**Issue by stojg** [stojg/purr#23]
Purr will currently filter out PRs that has a "needs changes" until it's closed. This means that commits done by the author after will show up in the purr listing. 

We could change this to compare times between "needs changes" and last commit

**Comment by stojg** [comment]
This might be unsolvable in a generic way since it's a lot about team workflow, it might be possible to implement this in the review filter, or add filter that expands on this feature.

**Comment by andrewandante** [comment]
Github has a "request changes" API that can be utilised for peer review, and we could hook in to that with Purr - see https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request

Adds a little bit of overhead, but would be better from a workflow POV too.

**Comment by stojg** [comment]
Yeah, it’s easy to filter “out” with that API and purr currently support it
(see filters in readme). The trick is how to signal back to to purr that a
PR is ready for review again.

Commits after the after “changes requested” might be to blunt depending on
the teams workflow.

**Comment by andrewandante** [comment]
Ah I see, excellent. That'll teach me to read the whole README.

Almost need some sort of filter syntax in the JSON so it can be organised into groups:

```JSON
"filters": [
        {
            "title": "Work in Progress",
            "show": {
                "Open": false,
                "WIP": true,
                "ChangesRequested": false,
                "HasCommitsAfterChangesRequested": false
            }
        },
        {
            "title": "Ready for Review",
            "show": {
                "Open": true,
                "WIP": false,
                "ChangesRequested": false,
                "HasCommitsAfterChangesRequested": true
            }
        },
        {
            "title": "Changes Requested",
            "show": {
                "Open": false,
                "WIP": false,
                "ChangesRequested": true,
                "HasCommitsAfterChangesRequested": false
            }
        }
    ]
```
